cmd/otel-agent/main.go

```diff
@@ -0,0 +1,94 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// TODO: docs
```

Comment:
Let's mention that this is for test purposes only here?

Reply:
Yes, I'd love to keep it for that.